packages/adapters/src/docker-sequence-adapter.ts

```diff
@@ -205,9 +205,15 @@ class DockerSequenceAdapter implements ISequenceAdapter {
         const engines = validPackageJson.engines ? { ...validPackageJson.engines } : {};
         const config = validPackageJson.scramjet?.config ? { ...validPackageJson.scramjet.config } : {};
 
+        const container = Object.assign({}, this.config.docker.runner);
+
+        container.image = "python3" in engines
+            ? this.config.docker.runnerImages.python3
+            : this.config.docker.runnerImages.node;
+
```

Comment:
This is just a suggestion, do you thing we should e.g. require "engines" key in all sequences, with no default to fall back to?

Reply:
ok. I thought so, too.